apps/src/code-studio/components/ImagePicker.jsx

```diff
@@ -10,10 +10,10 @@ import {ICON_PREFIX} from '@cdo/apps/applab/constants';
 const extensionFilter = {
   // Note: .jfif files will be converted to .jpg by the server.
   image: '.jpg, .jpeg, .jfif, .gif, .png',
-  audio: '.mp3',
+  audio: '.mp3, .wav',
   document: '.jpg, .jpeg, .gif, .png, .pdf, .doc, .docx',
   // Default set of valid extensions (used if type filter is not specified)
-  default: '.jpg, .jpeg, .jfif, .gif, .png, .mp3, .pdf, .doc, .docx'
+  default: '.jpg, .jpeg, .jfif, .gif, .png, .mp3, .wav, .pdf, .doc, .docx'
```

Comment:
(I know this is already merged but just out of curiosity) - is the starter assets manager a different asset manager than the standard one? Asking because when I ran into this for the standard level asset manager, I set it up so that we supplied a custom list of extensions (including .wav) so that we could explicitly include .wavs and not include .mp3s (source [here](https://github.com/code-dot-org/code-dot-org/blob/6c616357d384937e1d14bcc82321464d92ce0acb/apps/src/javalab/PreviewPaneHeader.jsx#L48)). I don't know how much that applies here but just wondering if those two asset managers are the same or different.

Reply:
Thank you for getting this fix out quickly! Can you confirm - does the original asset manager (on App Lab or Game Lab) now allow upload of `.wav` files? In the past, we wanted those labs to only support `.mp3` files for audio since that is the only type IE11 supports (happy to share more details there - the short story is we don't need that same restriction on Java Lab as we won't be supporting IE11 on Java Lab).

Reply:
Okay thank you both for the comments here! In short: here's what I've figured out. Yes, this affects the other labs as well, and I *think* we no longer care about this distinction because we aren't supporting ie11 in other labs anymore. Yes, there is a different asset manager for start sources specifically. I've let starlabs know and will revert this accordingly/if necessary.